conf/username.txt

```diff
@@ -1 +1 @@
-your-github-username-here-in-conf-file
+suja8742
```

Comment:
@suja8742 for the `.username.txt.swp` file above you should avoid committing that file.  You can add `*.swp` to your `.gitignore` file if you like.  Take a look at what you are committing and make sure you only add files you intend to add.